ext/msgpack/buffer.h

```diff
@@ -494,9 +494,9 @@ static inline VALUE msgpack_buffer_read_top_as_string(msgpack_buffer_t* b, size_
 #endif // HAVE_RB_ENC_INTERNED_STR
 }
 
-static inline VALUE msgpack_buffer_read_top_as_symbol(msgpack_buffer_t* b, size_t length)
+static inline VALUE msgpack_buffer_read_top_as_symbol(msgpack_buffer_t* b, size_t length, int utf8)
```

Comment:
`msgpack_buffer_read_top_as_string` defines the last argument as `bool`.
Is there any reason to use `int` here?

Reply:
Nope, just an oversight, I'll fix it right now.

Reply:
Fixed.